### PR TITLE
games-rpg/baldurs-gate-ee: bump to 2.5.23121

### PR DESCRIPTION
**Pull request by diogocp**

### CODE REVIEW

File: games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.5.23121.ebuild [hunk 1]
```diff
@@ -1,56 +1,53 @@
...
 RESTRICT="bindist fetch"
 
 DEPEND="app-arch/unzip"
-RDEPEND="dev-libs/expat[abi_x86_32(-)]
-	dev-libs/json-c[abi_x86_32(-)]
-	dev-libs/openssl:0[abi_x86_32(-)]
-	media-libs/openal[abi_x86_32(-)]
-	virtual/opengl[abi_x86_32(-)]
-	x11-libs/libX11[abi_x86_32(-)]"
+RDEPEND="dev-libs/expat
```
[the remark this answers is not in the document]
Comment (diogocp) [reply games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.5.23121.ebuild]: I don't think that is necessary. This new version includes both x86 and amd64 binaries.

File: games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.5.23121.ebuild [hunk 2]
```diff
@@ -1,56 +1,53 @@
 # Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-inherit eutils unpacker
+inherit desktop eutils unpacker
```
[the remark this answers is not in the document]
Comment (diogocp) [reply games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.5.23121.ebuild]: `eutils` is required for `make_wrapper`.

Without `unpacker` the file is not recognized. With `unpacker` but no explicit `unpack_zip`, unpacking fails. I could use `unzip` directly, but `unpack_zip` seems better. This was also [discussed before](https://github.com/gentoo/gentoo/pull/4928#discussion_r122597653).

[the remark this answers is not in the document]
Comment (a17r) [reply games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.5.23121.ebuild]: indeed, thanks.

File: games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.5.23121.ebuild (hunk 1 above)
[the remark this answers is not in the document]
Comment (a17r) [reply games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.5.23121.ebuild]: but then you need to make sure that the dependencies are built for these ABIs as well?

[the remark this answers is not in the document]
Comment (diogocp) [reply games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.5.23121.ebuild]: I don't want this package to have "multilib" support. It's an application, so I only want to install it for the native ABI. AFAIK the native ABI is implied in dependency specifications that do not explicitly specify another one.

[the remark this answers is not in the document]
Comment (a17r) [reply games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.5.23121.ebuild]: ah, yeah. I misinterpreted your previous message, but the ebuild code makes it clear anyway.

**Comment by diogocp** [comment]
I kept the old version for now.

**Pull request closed without merging**